Chapters/6.Qualified_identifiers.md

```diff
@@ -22,6 +22,8 @@ as detailed in the following sections.
 ## 6.1 Fragment qualifiers
 
 There are two fragment qualifiers, `lines` and `bytes`.
+Each fragment qualifier MUST appear at most once.
```

Comment:
Since we also say that both cannot be present, maybe we can re-word this to:

Each valid SWHID must have at most one fragment qualifier.

Reply:
Thanks, PR updated accordingly!